website/architecture/render-pipeline.md

```diff
@@ -189,7 +189,7 @@ After React creates the new _React Element Tree_ and _React Shadow Tree_, it mus
 - **Tree Promotion (Next Tree → Rendered Tree)**: This step atomically promotes the “next tree” to “previously rendered tree” so that the next mount phase computes a diff against the proper tree.
 - **Tree Diffing:** This step computes the diff between the “previously rendered tree” (**T**) and the “next tree” (**T'**). The result is a list of atomic mutation operations to be performed on _host views_.
   - In the above example, the operations consist of: `UpdateView(**Node 3**, {backgroundColor: 'yellow'})`
-    Diff can be calculated for any currently mounted tree with any new tree. The renderer can skip some intermediate versions of the tree.
+  - The diff can be calculated for any currently mounted tree with any new tree. The renderer can skip some intermediate versions of the tree.
```

Comment:
I'm not sure if this extra dot-point was the intention of the original writer - I'm just guessing, based on the new line.

The current text, without the extra dot-point, renders kind of weird. Looks like a mistake:

<img width="812" alt="Screenshot 2024-09-09 at 21 55 38" src="https://github.com/user-attachments/assets/99e60fd3-347a-40c8-9391-18169e0fc4f6">